src/txadmission.cpp

```diff
@@ -487,8 +496,9 @@ bool AcceptToMemoryPool(CTxMemPool &pool,
 
     bool isRespend = false;
     bool missingInputs = false;
+    bool fWaitQ = false;
     res = ParallelAcceptToMemoryPool(txHandlerSnap, pool, state, tx, fLimitFree, &missingInputs, fOverrideMempoolLimit,
-        fRejectAbsurdFee, allowedTx, vCoinsToUncache, &isRespend);
```

Comment:
If we let the caller choose to push the tx to the wait Q (see next comment), then we don't need this fWaitQ param...

Reply:
done, see above